ui/lib/kv/addon/templates/secret/metadata/index.hbs

```diff
@@ -1 +1,6 @@
-<Page::Secret::MetadataDetails @model={{this.model}} @breadcrumbs={{this.breadcrumbs}} />
\ No newline at end of file
+<Page::Secret::MetadataDetails
+  @model={{this.model.metadata}}
```

Comment:
Could this arg be a little more explicit? (After your feedback I updated the args for the details page as `@secret` and `@metadata`
```suggestion
  @metadata={{this.model.metadata}}
```